source/clear-linux/tutorials/flatpak/flatpak.rst

```diff
@@ -1,103 +1,127 @@
-.. _flatpak:
+.. _flatpak-tutorial:
 
-Use Flatpak\* to install applications on |CL-ATTR|
-##################################################
+Use Flatpak\* to install applications
+#####################################
 
-This tutorial provides all the required steps to install Flatpak as well as
-downloading, installing, and running LibreOffice\* on |CL-ATTR|.
+`Flatpak`_ is a framework for building and distributing desktop apps on
+Linux\*. It allows you to build a single app and install it on different
+distributions of Linux. Flatpak apps are available through `Flathub`_ or the
+`Clear Linux Store`_.
 
-Please visit the `Flatpak website`_ for more information about Flatpak and
-how to use it. You can also `download it here`_.
+This tutorial shows how to install a Flatpak app on |CL| using Gnome Software
+and the command line.
 
-Before you begin
-****************
+Prerequisites
+*************
 
-This tutorial assumes you have installed |CL| on your host system.
-For detailed instructions on installing |CL| on a bare metal system, visit
-our :ref:`bare metal installation guide <bare-metal-install-desktop>`.
+* |CL| installed on host system
 
-Install Flatpak on your host system
-===================================
+  Refer to :ref:`Get started <get-started>` for installation instructions.
 
-Flatpak is included as part of the bundle `desktop`. To install the
-application, log in to your user account and enter the following command:
+* `desktop` bundle installed
 
-.. code-block:: bash
+  Flatpak is included as part of the `desktop` bundle. The Flathub repository is

```

Comment:
Please revise this per @bktan8 comments in #481  
Add keyword to close that issue too.

Reply:
Hold pls @krisdale . @bktan8 please first respond to my comments on #481.

Reply:
@bktan8 thanks for your response. 
@krisdale please revise  command to reflect:  
`sudo swupd bundle-add desktop-autostart`.
Revise text to reflect above change, pls.
Revise first sentence of note: Flatpak is included via `desktop`, which is included in `desktop-autostart`.